sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_cbs_async.py

```diff
@@ -210,23 +225,23 @@ async def update_token(self):
                 extra=self._network_trace_params
             )
         self._expires_on = access_token.expires_on
-        expires_in = self._expires_on - int(utc_now().timestamp())
+        expires_in = cast(int, self._expires_on) - int(utc_now().timestamp())
```

Comment:
comment w/ reason for cast might be good

Reply:
removed the cast altogether :)